assets/app/view/game/corporation.rb

```diff
@@ -81,9 +81,11 @@ def render_title
           },
         }
 
+        full_name = @corporation.full_name
+        full_name += " (#{@corporation.extra_info})" unless @corporation.extra_info.nil?
```

Comment:
we don't need the extra info attribute, delete that, we should simply render abilities for corporations, if that's not clear let's chat about it on slack

Reply:
@tobymao: This is not done today, I suppose. Is the idea to show the name of all abilities that is not part of a company?

So it would be
_ABC         4   $70
            Trains Cash
...
Company                              Income
Memphis & Charle....                    $0

Ability
Historical Objective:     Birmingham_

Reply:
This is how the Entity page looks like with this feature.
![Screenshot from 2020-07-13 19-02-18](https://user-images.githubusercontent.com/2631151/87332950-b4305f00-c53c-11ea-9419-cff594e749c9.png)

Reply:
@tobymao Do you think this view is OK? 

In case of a corporation that owns both companies and have attributes the view is
![Corporation_with_ability_and_company](https://user-images.githubusercontent.com/2631151/87934915-58ac2700-ca90-11ea-8a77-303646a1e9fb.PNG)